commit/outcome_test.go

```diff
@@ -0,0 +1,44 @@
+package commit
+
+import (
+	"testing"
+
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_buildReport(t *testing.T) {
+	t.Run("determinism check", func(t *testing.T) {
```

Comment:
I feel like all these determinism checks would be good for fuzz tests, we can explore that in the future